.github/ISSUE_TEMPLATE/report-a-bug.md

```diff
@@ -0,0 +1,34 @@
+---
+name: report a bug
+about: Create a report to help us improve
+title: "[BUG] "
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug:**
+A clear and concise description of what the bug is.
```

Comment:
Since na majority of the bugs will relate to sql maybe explicitly name, that the sql statement needs to be added?

Reply:
![image](https://user-images.githubusercontent.com/58105072/175830526-a5e4c039-dc0d-48ed-88d6-07126b4f35ff.png)

changed to ↑